src/pages/index.tsx

```diff
@@ -102,11 +109,16 @@ export default Index;
 
 export const getServerSideProps: GetServerSideProps = withSSRLogger(
   async () => {
-    const fetchedData = await getIdentities();
+    const sortedIdentities = await getIdentities().then((result) => {
+      if (result instanceof Error) {
+        throw new Error();
```

Comment:
```suggestion
        throw new Error(result);
```

Reply:
OK done c8e792e, I had to stringify the result though, class Error was expecting an argument of type string not of type error

Reply:
Well, yes... since that's an instance of Error, we should just `throw result`, sorry 🙇‍♂️

Reply:
No prob! fixed 745ec61